docs/homestead_prov_api.md

```diff
@@ -236,6 +236,9 @@ Parameters:
     result in a faster response, but will be less well-paced (with a higher risk of disrupting
     service), and as Homestead handles more data at once, results in higher memory usage. The value
     of 256 has proved to work well in testing.
+* chunk (integer, default unset) - If set, this API only returns this chunk (i.e. a fraction of
```

Comment:
Might be worth a usage example - e.g. if you have 10,000 subscribers then `/public/?chunk-proportion=1000&chunk=0` returns ~10, `/public/?chunk-proportion=1000&chunk=1` returns the 'next' ~10 (but they're not alphabetical)...